src/GitVersion.Core/VersionCalculation/NextVersionCalculator.cs

```diff
@@ -9,81 +9,90 @@ namespace GitVersion.VersionCalculation;
 public class NextVersionCalculator : INextVersionCalculator
 {
     private readonly ILog log;
-    private readonly IBaseVersionCalculator baseVersionCalculator;
+
     private readonly IMainlineVersionCalculator mainlineVersionCalculator;
+
     private readonly IRepositoryStore repositoryStore;
-    private readonly IIncrementStrategyFinder incrementStrategyFinder;
+
     private readonly Lazy<GitVersionContext> versionContext;
+    private GitVersionContext context => this.versionContext.Value;
```

Comment:
Properties should have uppercase first letters even though they are private.

```suggestion
    private GitVersionContext Context => this.versionContext.Value;
```

Reply:
Totally agree. Done